.github/workflows/helm-test.yml

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+name: Lint and Test Charts
+
+on:
+    pull_request:
+    workflow_dispatch:
+        inputs:
+            node_image:
+                description: 'kindest/node image for k8s kind cluster'
+                # k8s version from 3.1 release as default
+                default: 'kindest/node:1.27.3'
```

Comment:
Issue due to a typo in the image tag (lost 'v')  hence unable to download it.

Use: kindest/node:v1.27.3

Reply:
Thanks for pointing out! fixed